### PR TITLE
[WIP] Sample Quart integration

### PR DESCRIPTION
**Pull request by grigi**
Sample Quart integration

* [x] Build a more sensible example
* [x] Build a small plugin that does tear-down automatically
* [x] Add it to documentation
* [x] Add CLI command `generate-schemas`
* [ ] Ensure we solve #119

### CODE REVIEW

File: examples/quart_example.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+@app.after_serving
+async def close_orm():
+    await Tortoise.close_connections()
+
+
+@app.route("/")
+async def list_all():
+    users, workers = await asyncio.gather(Users.all(), Workers.all())
+    return json.dumps(
```
Comment (pgjones) [inline examples/quart_example.py]: Quart has a `jsonify` function that should be used instead, as it sets the content-type header to json.

**Comment by pgjones** [comment]
It tortoise asyncio only or can Trio be used? If the latter is possible it might be nice to provide an example that could work with either (quart-trio also exists).

Thanks for adding a Quart example 🎉

File: examples/quart_example.py (hunk 1 above)
Comment (grigi) [reply examples/quart_example.py]: Thanks, will change.

**Comment by grigi** [comment]
I doubt Trio will work anytime soon. We use asyncio directly in a few places, and I suspect we may run into issues with the DB drivers.

**Comment by grigi** [comment]
@pgjones Could you please re-review, specifically look at: `examples/quart/` and `tortoise/contrib/quart/`
https://github.com/tortoise/tortoise-orm/pull/121/files#diff-059e35c4f26307fa382f487d0388a432
https://github.com/tortoise/tortoise-orm/pull/121/files#diff-f719a1e73bcf968f213cff7a2c990ce3
https://github.com/tortoise/tortoise-orm/pull/121/files#diff-7326eaad88442bfed6b2cbc3e7a3a986

Thanks in advance

**Comment by pgjones** [comment]
👍 Looks good to me.

**Pull request merged**